unity/src/main/java/io/delta/unity/UCCatalogManagedClient.java

```diff
@@ -163,6 +211,17 @@ private void validateLoadTableVersionExists(
     }
   }
 
+  private Map<String, String> getRequiredTablePropertiesForCreate(String ucTableId) {
+    final Map<String, String> requiredProperties = new HashMap<>();
+
+    requiredProperties.put(
+        CATALOG_MANAGED_ENABLEMENT_PROPERTY, TableFeatures.SET_TABLE_FEATURE_SUPPORTED_VALUE);
+
+    // TODO: use the ucTableId
```

Comment:
what will this be used for?

Reply:
Just something UC requires! `delta-spark` may sometimes poke at this.